main.go

```diff
@@ -49,6 +49,23 @@ var Options struct {
 	// this will be used for API calls back to the Assisted Service API
 	// Will default to the value held in HTTPS_CA_FILE unless overridden
 	AssistedServiceApiTrustedCAFile string `envconfig:"ASSISTED_SERVICE_API_TRUSTED_CA_FILE" default:""`
+
+	// OSImageDownloadHeaders contains a JSON encoded representation of any
+	// HTTP headers to be sent with every request to download an OS image.
+	OSImageDownloadHeaders string `envconfig:"OS_IMAGES_HTTP_REQUEST_HEADERS" default:""`
```

Comment:
Maybe the env var name should match the variable name here?

Reply:
Done